rustainers/src/runner/options.rs

```diff
@@ -31,7 +31,7 @@ pub struct RunOption {
 
     /// The network
     #[builder(default, setter(into))]
-    pub(crate) network: Network,
+    pub(crate) network: Option<Network>,
```

Comment:
Maybe exposing a public field is not the best option in this case

Reply:
It has impact on what was done before. I will need to double check with @IgorLaborieWefox

Reply:
I think that OK